### ISSUE TITLE
BadMigrationError with Django==1.8.6

### ISSUE THREAD

**Issue by iiman** [mytardis/mytardis#557]
Hi

The following error occurs when deploying MyTardis since the release of Django 1.8.6 on November 4th. 

**django.db.migrations.loader.BadMigrationError: Migrated app 'django_extensions' contains South migrations. Make sure all numbered South migrations are deleted prior to creating Django migrations.**

Since MayTardis's Django requirement is Django>=1.8,<1.9, the most recent Django (1.8.6) is downloaded, which causes the error.

As the workaround, I am downgrading to 1.8.5. It would be great if either the Django requirement is set to 1.8.5 or the issue with Django==1.8.6 is fixed.

Cheers,

**Comment by grischa** [comment]
Are you running the latest develop and did you run `pip install -U -r requirements.txt`?

**Comment by iiman** [comment]
I am running develop branch (3.7+); SHA = 8fbb93bfc07934cc65f5d51bb9dbd44fc4cb1b16

I used the following commands:

pip install -U pip
pip install -r requirements.txt

**Comment by grischa** [comment]
Which version of django-extensions is installed? `pip show django-extensions`

**Comment by iiman** [comment]
Metadata-Version: 2.0

Name: django-extensions

Version: 1.1.1

Summary: Extensions for Django

Home-page: http://github.com/django-extensions/django-extensions

Author: Bas van Oostveen

Author-email: v.oostveen@gmail.com

License: New BSD License

Location: /usr/local/lib/python2.7/dist-packages

Requires: six


On 12 November 2015 at 12:11, Grischa Meyer <notifications@github.com>
wrote:

> Which version of django-extensions is installed? pip show
> django-extensions
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mytardis/mytardis/issues/557#issuecomment-155965709>.
>



-- 
Iman I. Yusuf, PhD
Software Engineer and Research Academic, E-Research Office
Research & Innovation Portfolio
RMIT University, Melbourne, Australia

Phone:     +61 3 9925 0114
Location:  96.1.5
<https://plus.google.com/100700838391826629572/about>
<http://au.linkedin.com/in/imanyusuf>  <https://twitter.com/iimanyusuf>

**Comment by grischa** [comment]
Thanks!
Can you please run `pip install -U -r requirements.txt`? That should fix the issue.

**Comment by iiman** [comment]
Hi Grischa,

Actually when I run `pip install -U -r requirements.txt`, the installation completes successfully. However, MyTardis exits with status 0. One or more packages that are upgraded could be causing this issue. I did not have the time to look into this closely.

I did the following to fix the problem. 
1) Since I am running the `develop` branch whose head is at commit  8fbb93bfc07934cc65f5d51bb9dbd44fc4cb1b16, I replaced the `requirements.txt` of 8fbb93bfc07934cc65f5d51bb9dbd44fc4cb1b16 by the latest `requirements.txt` commit (bc08e568105756aa5ebcccbfefba9a7f92e2dc40). The latest `requirements.txt` doesn't specify the versions of `django-celery, django-extensions` and `kombu` packages. It also installs 'pillow'.
2) I also installed `libjpeg-dev` before installing the packages. `libjpeg-dev` is required for the successful installation of `pillow`

Now MyTardis installs and runs without any issue.

Cheers,
